.stylelintrc.json

```diff
@@ -1,11 +1,6 @@
 {
-  "extends": [
-    "stylelint-config-standard"
-  ],
-  "plugins": [
-    "stylelint-scss",
-    "stylelint-csstree-validator"
-  ],
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
```

Comment:
Kindly avoid modifying the linter configurations, they represent coding standards you share with your team.

Reply:
I understand what you mean (after carefully going through the code I realized the configuration was still the same)